.github/pull_request_template.md

```diff
@@ -0,0 +1,50 @@
+<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][OSDOCS-<jira-issue-id>]: <short-description-of-the-pr> --->
+
+Version(s):
+<!--- Specify the version or versions of OpenShift your PR applies to.
+Open your PR against the `main` branch for if your changes apply to the latest releases. --->
+
+Issue:
+<!--- Add a link to the Bugzilla, Jira, or GitHub issue. --->
+
+Link to docs preview:
+<!--- Add direct link(s) to the exact page(s) with updated content from the preview build.
+  - The preview will be generated after you open the PR.
+  - You will need to edit the comment to add the link after the preview builds.
+  - Review the preview build to make sure your changes are rendering properly. --->
+
+Additional information:
+<!--- Optional: Include additional context or expand the description here.
+
+Next steps after opening your PR:
+
+* Ask for peer review from the OpenShift docs team:
+  - For Red Hat associates: Ping the peer review squad (@peer-review-squad) requesting a review in the #forum-docs-review channel (CoreOS Slack workspace).
```

Comment:
Not important, but could trim this down to `Ping @peer-review-squad requesting a review...` and probably not lose any meaning.